timesketch/frontend-ng/src/components/LeftPanel/SigmaRule.vue

```diff
@@ -19,36 +19,92 @@ limitations under the License.
+                  <v-list-item-icon>
+                    <v-icon>mdi-archive</v-icon>
+                  </v-list-item-icon>
+                  <v-list-item-content>
+                    <v-list-item-title>EditRule with function</v-list-item-title>
+                  </v-list-item-content>
+                </v-list-item>
+
+                <v-list-item v-on:click="deprecateSigmaRule(sigmaRule.rule_uuid)">
+                  <v-list-item-icon>
+                    <v-icon>mdi-archive</v-icon>
+                  </v-list-item-icon>
+                  <v-list-item-content>
+                    <v-list-item-title>Deprecated Sigma rules will not be used in the Sigma analyzer</v-list-item-title>
+                  </v-list-item-content>
+                </v-list-item>
+                <v-list-item v-on:click="deleteRule(sigmaRule.rule_uuid)">
+                  <v-list-item-icon>
+                    <v-icon>mdi-delete</v-icon>
+                  </v-list-item-icon>
+                  <v-list-item-content>
+                    <v-list-item-title>Delete Rule</v-list-item-title>
+                  </v-list-item-content>
+                </v-list-item>
+              </v-list-item-group>
+            </v-list>
+          </v-card>
+        </v-menu>
+      </div>
     </v-row>
 
     <v-expand-transition>
       <div v-show="expanded" class="pa-4 pt-0">
-        <div style="font-size: 0.9em" class="mt-2">
+        <!--{{ sigmaRuleSummary }}-->
+        <div class="mt-2">
           <v-simple-table dense>
             <template v-slot:default>
               <tbody>
-                <tr v-for="(v, k) in sigmaRuleSummary" :key="v.id">
+                <tr v-for="(value, key) in sigmaRuleSummary" :key="value.rule_uuid">
                   <td>
-                    <strong>{{ k | capitalize }}</strong>
+                    <strong>{{ key }}</strong>
                   </td>
                   <td>
-                    <span v-if="k === 'references'">
-                      <div v-for="ref in v" :key="ref">
+                    <span v-if="key === 'references'">
+                      <div v-for="ref in value" :key="ref">
                         <a :href="ref" target="new">{{ ref }}</a>
                       </div>
                     </span>
-                    <span v-else-if="k === 'falsepositives'">
-                      <v-chip v-for="tag in v" :key="tag" rounded x-small class="mr-2">{{ tag }}</v-chip>
+                    <span v-else-if="key === 'falsepositives'">
+                      <v-chip v-for="falsepositive in value" :key="falsepositive" rounded x-small class="mr-2">{{
+                        falsepositive
+                      }}</v-chip>
                     </span>
-                    <span v-else-if="k === 'tags'">
-                      <v-chip v-for="tag in v" :key="tag" rounded x-small class="mr-2">{{ tag }}</v-chip>
+
+                    <span v-else-if="key === 'tags'">
```

Comment:
Check for `value` as well, and don't render if empty (otherwise you get an empty v-chip)

Reply:
done, PTAL